src/package/package.ts

```diff
@@ -194,6 +208,64 @@ export class Package {
     return convertPackage(pkgId, connection, options, project);
   }
 
+  /**
+   * Download the metadata files for a previously published package version, convert them to source format, and put them into a new project folder within the sfdx project.
+   *
+   * @param project
+   * @param options {@link PackageVersionMetadataDownloadOptions}
+   * @param connection
+   * @returns
+   */
+  public static async downloadPackageVersionMetadata(
+    project: SfProject,
```

Comment:
Does this really need a project?  Doesn't it just need an org connection and a 04t ID (and destination dir)?

Reply:
At the moment technically it does not but we expect we'll want to introduce some quality of life enhancements that update `sfdx-project.json` after downloading the metadata (to better support the happy path use case I described in an earlier comment).